From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Thu, 12 Oct 2017 08:15:17 +0200
Subject: Disable NTP remote suggestions by default

Disables Google "Snippets" & Articles from displaying on the New Tab Page.

Initial Commit: https://github.com/uazo/cromite/commit/a772e673180283f998c1a1fe0160752acf771734 (Inherited from Bromite)
Category: Privacy
License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 components/feed/core/shared_prefs/pref_names.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/components/feed/core/shared_prefs/pref_names.cc b/components/feed/core/shared_prefs/pref_names.cc
--- a/components/feed/core/shared_prefs/pref_names.cc
+++ b/components/feed/core/shared_prefs/pref_names.cc
@@ -24,8 +24,8 @@ const char kArticlesListVisible[] = "ntp_snippets.list_visible";
 const char kEnableSnippetsByDse[] = "ntp_snippets_by_dse.enable";
 
 void RegisterFeedSharedProfilePrefs(PrefRegistrySimple* registry) {
-  registry->RegisterBooleanPref(kEnableSnippets, true);
-  registry->RegisterBooleanPref(kArticlesListVisible, true);
+  registry->RegisterBooleanPref(kEnableSnippets, false);
+  registry->RegisterBooleanPref(kArticlesListVisible, false);
   registry->RegisterBooleanPref(kEnableSnippetsByDse, true);
 }
 
--
